src/EditorFeatures/XunitHook/XunitDisposeHook.cs

```diff
@@ -18,7 +18,7 @@ public void Execute()
         if (!AppDomain.CurrentDomain.IsDefaultAppDomain())
             throw new InvalidOperationException();
 
-        var xunitUtilities = AppDomain.CurrentDomain.GetAssemblies().Where(static assembly => assembly.GetName().Name.StartsWith("xunit.runner.utility")).ToArray();
+        var xunitUtilities = AppDomain.CurrentDomain.GetAssemblies().Where(static assembly => assembly.GetName().Name.StartsWith("xunit.runner.visualstudio")).ToArray();
```

Comment:
Should we fail if this assembly doesn't exist to catch this break sooner in the future?